html/changelogs/Banthebantz - Genetics revert.yml

```diff
@@ -0,0 +1,36 @@
+################################
```

Comment:
The CL stuff in your PR first comment, a bot will make a changelog based on what's between Cl and /Cl so you don't need to make the yml files anymore